source/includes/_payouts.md

```diff
@@ -376,6 +376,8 @@ requests with the same token signature will be rejected.
 By default in our [sandbox environment](https://api.sandbox.transferwise.tech) strong customer authentication is 
 disabled. You can enable it for your account on the [public keys management page](https://api.sandbox.transferwise.tech/public-keys).
 
+The option for toggling the check yourself will be available in production as well as long as it is optional.
```

Comment:
It was hard for me to read this triple `as`, what do you think about loosing one:
```suggestion
The option for toggling the check yourself will also be available in production as long as it is optional.
```